nmigen_boards/quickfeather.py

```diff
@@ -0,0 +1,107 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.quicklogic import *
+from nmigen_boards.resources import *
+
+
+__all__ = ["QuickfeatherPlatform"]
+
+
+class QuickfeatherPlatform(QuicklogicPlatform):
+    device      = "ql-eos-s3_wlcsp"
+    part        = "PU64"
+    default_clk = "clk"
+    default_rst = None
+    speed = None
+    connectors = []
+    resources   = [
+        Resource("clk", 0, Pins("63", dir="i"), Clock(10e6), Attrs(IOSTANDARD="LVCMOS33")),
+
+        *ButtonResources(pins="62", attrs=Attrs(IOSTANDARD="LVCMOS33")),
+
+        RGBLEDResource(0, r="34", g="39", b="38", attrs=Attrs(IOSTANDARD="LVCMOS33")),
+
+        UARTResource(0,
+            rx="9", tx="8",
+            attrs=Attrs(IOSTANDARD="LVCMOS33")
+        ),
+
+        Resource("spi_master", 0,
+            Subsignal("clk", Pins("20", dir="i")),
+            Subsignal("miso", Pins("17", dir="o")),
+            Subsignal("mosi", Pins("16", dir="i")),
+            Subsignal("ss", Pins("11", dir="i")),
+            Subsignal("cs2", Pins("28", dir="i")),
+            Subsignal("cs3", Pins("18", dir="i")),
+            Attrs(IOSTANDARD="LVCMOS33")
+        ),
+
+        Resource("spi_slave", 0,
+            Subsignal("clk", Pins("40", dir="i")),
+            Subsignal("miso", Pins("42", dir="o")),
+            Subsignal("mosi", Pins("36", dir="i")),
+            Subsignal("cs_n", Pins("37", dir="i")),
+            Attrs(IOSTANDARD="LVCMOS33")
+        ),
+
+        Resource("i2c", 0,
```

Comment:
These two resources would be completely unusable because the signal directions are wrong.